db/migrate/20150608145958_remove_unused_topic_record.rb

```diff
@@ -1,36 +0,0 @@
-class RemoveUnusedTopicRecord < ActiveRecord::Migration
-  def up
-    remove_subtopic
-    redirect_former_url
-  end
-
-  def down
-  end
-
-  def remove_subtopic
-    subtopic ||= Topic.find_by(content_id: "4ad3f330-8041-4960-826b-54327b4026ca")
-    subtopic.destroy if subtopic
-  end
-
-  def redirect_former_url
-    publishing_api = CollectionsPublisher.services(:publishing_api)
-    resp = publishing_api.put_content_item("/immigration-operational-guidance/european-casework-instructions", redirect_item)
-    puts resp.body
-  end
-
-
-  def redirect_item
-    {
-      "format" => "redirect",
-      "publishing_app" => "collections-publisher",
-      "update_type" => "major",
-      "redirects" => [
-        {
-          "path" => "/immigration-operational-guidance/european-casework-instructions",
-          "type" => "exact",
-          "destination" => "/immigration-operational-guidance",
-        },
-      ],
-    }
```

Comment:
I'm not sure we should remove this migration just yet.  It's the only place that this redirect is recorded.  In the future we will likely support archiving/renaming tags, at which point this can be modled in the database.